### PR TITLE
refactor: log publicPath info

### PR DESCRIPTION
**Pull request by snitin315**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [x] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

For https://github.com/webpack/webpack-dev-server/pull/3218

### Breaking Changes
Maybe

### Additional Info
None

### CODE REVIEW

File: src/utils/getPaths.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ export default function getPaths(context) {
       ? compilation.getPath(compilation.outputOptions.publicPath)
       : '';
 
+    if (publicPath) {
+      context.logger.info(`webpack output is served from "${publicPath === 'auto' ? '/' : publicPath}" URL`);
+    }
```
Comment (alexander-akait) [inline src/utils/getPaths.js:17]: It will be spam on each request, we need move it only on constructor place

Comment (snitin315) [reply src/utils/getPaths.js:17]: I will update 👍🏻

**Comment by alexander-akait** [comment]
I was wrong, I think we don't need this logging, it is weird, because `publicPath` can be `Function` or/and can contain `[hash]`/`[fullhash]`, so our logging will be weird or invalid, I think docs are enough

**Comment by snitin315** [comment]
So we just remove this logging from dev-server ?

**Comment by alexander-akait** [comment]
Yes, I think most of developers see these messages like spam, so let's remove it, I don't think we create problem here removing this, also plugins can change publicPath too, so we know real publicPath only after compilation

**Comment by snitin315** [comment]
Makes sense 👍

**Pull request closed without merging**